src/WebEid.AspNetCore.Example.Tests/UnitTest1.cs

```diff
@@ -0,0 +1,18 @@
+using NUnit.Framework;
+
+namespace WebEid.AspNetCore.Example.Tests
+{
+	public class Tests
```

Comment:
What's with the test, do you plan to implement it?

Reply:
Nope :)